SPECS/pango/pango.spec

```diff
@@ -1,45 +1,50 @@
-Summary:	library for laying out and rendering of text.
-Name:		pango
-Version:	1.40.4
-Release:    4%{?dist}
-License:	LGPLv2 or MPLv1.1
-URL:		http://pango.org
-Group:		System Environment/Libraries
+Summary:        library for laying out and rendering of text.
+Name:           pango
+Version:        1.40.4
+Release:        5%{?dist}
+License:        LGPLv2 or MPLv1.1
+URL:            http://pango.org
+Group:          System Environment/Libraries
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	    https://download.gnome.org/sources/pango/1.40/%{name}-%{version}.tar.xz
-BuildRequires:	glib-devel
-BuildRequires:	cairo
-BuildRequires:	cairo-devel
-BuildRequires:	libpng-devel
-BuildRequires:	fontconfig
-BuildRequires:	fontconfig-devel
-BuildRequires:	harfbuzz
-BuildRequires:	harfbuzz-devel
-BuildRequires:	freetype
-Requires:	    harfbuzz-devel
+Source0:        https://download.gnome.org/sources/pango/1.40/%{name}-%{version}.tar.xz
+BuildRequires:  glib-devel
+BuildRequires:  cairo
+BuildRequires:  cairo-devel
+BuildRequires:  libpng-devel
+BuildRequires:  fontconfig
+BuildRequires:  fontconfig-devel
+BuildRequires:  harfbuzz
+BuildRequires:  harfbuzz-devel
+BuildRequires:  freetype
+Requires:       harfbuzz-devel
+
 %description
 Pango is a library for laying out and rendering of text, with an emphasis on internationalization. Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the context of the GTK+ widget toolkit.
 
-%package	devel
-Summary:	Header and development files
-Requires:	%{name} = %{version}-%{release}
-%description	devel
+%package    devel
+Summary:    Header and development files
+Requires:   %{name} = %{version}-%{release}
+
+%description    devel
 It contains the libraries and header files to create applications
 
 %prep
 %setup -q
+
 %build
 ./configure \
-	--prefix=%{_prefix}
+    --prefix=%{_prefix}
 make %{?_smp_mflags}
+
 %install
 make DESTDIR=%{buildroot} install
 find %{buildroot} -name '*.la' -delete
 
 %check
-#These tests are known to fail. Hence sending exit 0
-make %{?_smp_mflags} -k check || exit 0
+# Skip test-layout test, which is known to fail
```

Comment:
It is known to fail for us or in general and there's simply no upstream fix?

Reply:
It's known to fail in general when the system doesn't have Cantarell fonts like ours. See note on http://www.linuxfromscratch.org/blfs/view/svn/x/pango.html